src/Selenium2Library/keywords/_element.py

```diff
@@ -297,6 +297,18 @@ def get_horizontal_position(self, locator):
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+
+        The element width and height is returned.
+        Fails if a matching element is not found.
+        """
+        element = self._element_find(locator, True, False)
```

Comment:
It could be useful to put the ```required``` argument as ```True```. In that way one does not need the lines 307 and 308. Also it quarentees unified error message.